content/influxdb/v2.0/write-data/delete-data.md

```diff
@@ -48,42 +49,84 @@ Delete data from buckets you've created. You cannot delete data from system buck
-{{% /note %}}
+{{% warn %}}
+Deleting data without a [delete predicate](/influxdb/v2.0/reference/syntax/delete-predicate)
+deletes all data in the specified bucket with timestamps between the specified `start` and `stop` times.
+{{% /warn %}}
 
 ## Delete data using the influx CLI
 
+{{% note %}}
+Use [InfluxDB CLI connection configurations](/influxdb/v2.0/reference/cli/influx/config/)
+to provide your **InfluxDB host, organization, and authentication token**.
+{{% /note %}}
+
 1. Use the [`influx delete` command](/influxdb/v2.0/reference/cli/influx/delete/) to delete points from InfluxDB.
-2. Specify your organization, bucket, and authentication token.
-3. Define the time range to delete data from with the `--start` and `--stop` flags.
+2. Use the `--bucket` flag to specify which bucket to delete data from.
+3. Use the `--start` and `--stop` flags to define the time range to delete data from.
+   Use [RFC3339 timestamps](/influxdb/v2.0/reference/glossary/#rfc3339-timestamp).
+4. _(Optional)_ Use the `-p`, `--predicate` flag to include a [delete predicate](/influxdb/v2.0/reference/syntax/delete-predicate)
+   that identifies which points to delete.
 
-#### Example
+### Examples
 
+##### Delete points in a specific measurement with a specific tag value
 ```sh
-influx delete -o my-org -b my-bucket -t $INFLUX_TOKEN \
- --start '1970-01-01T00:00:00.00Z' \
- --stop '2020-01-01T00:00:00.00Z' \
+influx delete --bucket example-bucket \
+  --start '1970-01-01T00:00:00Z' \
+  --stop $(date +"%Y-%m-%dT%H:%M:%SZ") \
+  --predicate '_measurement="example-measurement" AND exampleTag="exampleTagValue"'
 ```
 
-## Delete data using the API
-
-1. Use the InfluxDB API `/delete` endpoint to delete points from InfluxDB.
-2. Include your organization and bucket as query parameters in the request URL.
-3. Use the `Authorization` header to provide your InfluxDB authentication token.
-4. In your request payload, define the time range to delete data from with `start` and `stop`.
+##### Delete all points in a specified time range
+```sh
+influx delete --bucket example-bucket \
+  --start 2020-03-01T00:00:00Z \
+  --stop 2020-11-14T00:00:00Z
+```
 
-#### Example
+## Delete data using the API
+Use the InfluxDB API `/api/v2/delete` endpoint to delete points from InfluxDB.
+Include the following:
+
+- **Request method:** `POST`
+- **Headers:**
+  - **Authorization:** `Token` schema with your InfluxDB authentication token
+  - **Content-type:** `application/json`
+- **Query parameters:**
+  - **org** or **orgID:** organization name or [organization ID](/influxdb/v2.0/organizations/view-orgs/#view-your-organization-id)
+  - **bucket** or **bucketID:** bucket name or [bucket ID](/influxdb/v2.0/organizations/buckets/view-buckets/)
+- **Request body:** JSON object with the following fields:  
+  {{< req type="key" >}}
```

Comment:
This is definitely up for discussion. Everything else is required. I structured it this way because not every key in the Request body is required. Should I just flag everything in the list as required?